internal/validation/validation_test.go

```diff
@@ -33,22 +33,22 @@ func TestAll(t *testing.T) {
 		Tests   []*Test
 	}
 	if err := json.NewDecoder(f).Decode(&testData); err != nil {
-		t.Fatal(err)
+		t.Fatalf("test data JSON decode failed: %v", err)
 	}
 
 	schemas := make([]*schema.Schema, len(testData.Schemas))
 	for i, schemaStr := range testData.Schemas {
 		schemas[i] = schema.New()
 		if err := schemas[i].Parse(schemaStr); err != nil {
-			t.Fatal(err)
+			t.Fatalf("schema parse failed: %v", err)
```

Comment:
With the additional test cases, schema parsing fails because the schema definition language (SDL) has added the `&` symbol in describing [interface implementations](http://facebook.github.io/graphql/draft/#sec-Interfaces).

```graphql
interface NamedEntity {
  name: String
}

interface ValuedEntity {
  value: Int
}

type Person implements NamedEntity {
  name: String
  age: Int
}

type Business implements NamedEntity & ValuedEntity {
  name: String
  value: Int
  employeeCount: Int
}
```